test/parallel_api/algorithm/alg.modifying.operations/transform_binary.pass.cpp

```diff
@@ -107,7 +107,7 @@ test(Predicate pred, _IteratorAdapter adap = {})
 
         invoke_on_all_policies<CallNumber>()(test_one_policy(), adap(in1.begin()), adap(in1.end()), adap(in2.begin()),
                                              adap(in2.end()), adap(out.begin()), adap(out.end()), pred);
-        invoke_on_all_policies<CallNumber + 1>()(test_one_policy(), adap(in1.cbegin()), adap(in1.cend()),
+        invoke_on_all_policies<CallNumber + 10>()(test_one_policy(), adap(in1.cbegin()), adap(in1.cend()),
```

Comment:
Why exactly `+10` ?

Reply:
10 looks enough. Even if some body will add 1-5 additional calls (with the same other types) it works. No problem to write bigger 'capacity' - let's write 100, if we think that it is not ebough.

Reply:
If we are ready to discuss about this, so let's discuss.
From my point of view, better to manage this things on caller side, and have simple and clear solution as before.

Reply:
I've just proposed 15 line to change instead of 24 lines in Sergey's variant.
Both variant are absolutely the same. You can choose any one. I have no objections.

Reply:
... OK, there are some issues with CI with the second variant. So, I would suggest dont waste time and chose the first proposed variant.

Reply:
In this case you are able simple to revert one top commit.